hazelcast/src/main/java/com/hazelcast/collection/impl/queue/operations/QueueOperation.java

```diff
@@ -99,6 +98,7 @@ public void afterRun() throws Exception {
 
     @Override
     public void beforeRun() throws Exception {
+        container = getOrCreateContainer();
```

Comment:
Could this container 'resetting' by an issue with other operations like Lock/CountdownLatch etc?